src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +107,54 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={onInputChange}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {onError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames('is-light', 'button',
+                { 'is-loading': isLoading })}
+              disabled={query.trim() === ''}
```

Comment:
One of the possible ways. Just my suggestion. 
```suggestion
              disabled={!query.trim().length}
```